subtests/docker_cli/selinux_labels/test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# FIXME: This should be re-implemented in the subtest code (in python).
+#        This script is just a short-term solution.
+
+set -ex
```

Comment:
Eeek, the -x makes it painful to read error output. And the -e is a bad idea: it causes silent failure on unexpected circumstances (like failure in ps). Would you consider evaluating my rewrite?